lnrpc/rpc.proto

```diff
@@ -2659,7 +2659,8 @@ message Invoice {
 
     /*
     Route hints that can each be individually used to assist in reaching the
-    invoice's destination.
+    invoice's destination. The specified routing hints will be included in
+    the invoice.
```

Comment:
Comment changes need to re-generate protos as the comments are replicated in other `lnrpc` files

Reply:
Removed the comment change, not needed after all.